app/controllers/projects/samples_controller.rb

```diff
@@ -109,6 +118,53 @@ def destroy # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
       end
     end
 
+    def new_destroy_multiple
+      authorize! @project, to: :destroy_sample?
+      render turbo_stream: turbo_stream.update('samples_dialog',
+                                               partial: 'delete_multiple_samples_dialog',
+                                               locals: {
+                                                 open: true
+                                               }), status: :ok
+    end
+
+    def destroy_multiple # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
```

Comment:
Can we break this action down into multiple methods to get rid of the rubocop disables?

Reply:
Added in f692a237ab4fe9c1633b2388cae25ebea5929bbf